.github/workflows/selenium.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow will run headful selenium tests.
+
+name: Selenium tests
+
+# Declare default permissions as read only.
+permissions: read-all
+
+on:
+  merge_group:
+  pull_request:
+  push:
+    branches: 'main'
+  workflow_dispatch:
+    inputs:
+      verbose:
+        description: Verbose logging
+        default: false
+        required: false
+        type: boolean
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  selenium:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
```

Comment:
Unless there's a plan to run it in more OSes, `matrix:` can be removed.

Reply:
Let's keep it in case we want to extend run to other platforms